### PR TITLE
Feat/navbar

### PR DESCRIPTION
**Pull request by JumboDaniel**

### CODE REVIEW

**Comment by amjedidiah** [comment]
@JumboDaniel 
Please do the following for this branch:

1. Update this branch with code from `ft/landing` by running the following commands:

   - `git checkout ft/landing`
   - `git pull`: ensure no errors here
   - `git checkout feat/navbar`
   - `git merge ft/landing`: please reach out to me for any merge conflicts
2. Update this branch to only include code and assets for the Navbar
3. Fix issues with comments

File: src/pages/index.tsx [hunk 1]
```diff
@@ -14,8 +17,10 @@ export default function Home() {
           content="Dr Passy Amaraegbu is a father, psychologist and pastor specializing in solving long standing issues in the lives of people"
         />
       </Head>
-
       <Jumbo />
+      <Navbar/>
```
Comment (amjedidiah) [inline src/pages/index.tsx:21]: @JumboDaniel Please only include `Navbar` component

File: src/utils/theme.util.ts [hunk 2]
```diff
@@ -24,6 +24,8 @@ const initTheme = {
     black: "#1a1a1a",
     white: "#ffffff",
     grey2: "#808080",
+    lightGrey:"#E6E6E6",
```
Comment (amjedidiah) [inline src/utils/theme.util.ts:27]: Please remove Lines 27 and 28, as this is not meant for Navbar sectio

File: src/icons.tsx [hunk 3]
```diff
@@ -62,3 +62,80 @@ export const StarIcon = () => (
...
+        fill="#1A1A1A"
+      />
+      <path
+        d="M33.4172 10.8043L23.4338 4.25426C21.6338 3.07093 18.6672 3.07093 16.8672 4.25426L6.83383 10.8043C3.61716 12.8876 3.61716 17.6043 6.83383 19.7043L9.50049 21.4376L16.8672 26.2376C18.6672 27.4209 21.6338 27.4209 23.4338 26.2376L30.7505 21.4376L33.0338 19.9376V25.0376C33.0338 25.7209 33.6005 26.2876 34.2838 26.2876C34.9672 26.2876 35.5338 25.7209 35.5338 25.0376V16.8376C36.2005 14.6876 35.5172 12.1876 33.4172 10.8043Z"
+        fill="#1A1A1A"
+      />
+    </svg>
+  );
+};
+export const RoundStarIcon = () => {
```
Comment (amjedidiah) [inline src/icons.tsx:87]: Please remove this, as this is not meant for Navbar section

File: src/icons.tsx [hunk 4]
```diff
@@ -62,3 +62,80 @@ export const StarIcon = () => (
...
+        fill="#1A1A1A"
+      />
+      <path
+        d="M24.7958 22.9001L23.7625 24.1668C23.5958 24.3501 23.4792 24.7168 23.4958 24.9668L23.5958 26.6001C23.6625 27.6001 22.9458 28.1168 22.0125 27.7501L20.4958 27.1501C20.2625 27.0668 19.8625 27.0668 19.6292 27.1501L18.1125 27.7501C17.1792 28.1168 16.4625 27.6001 16.5292 26.6001L16.6292 24.9668C16.6458 24.7168 16.5292 24.3501 16.3625 24.1668L15.3292 22.9001C14.6792 22.1334 14.9625 21.2834 15.9292 21.0334L17.5125 20.6334C17.7625 20.5668 18.0625 20.3334 18.1958 20.1168L19.0792 18.7501C19.6292 17.9001 20.4958 17.9001 21.0458 18.7501L21.9292 20.1168C22.0625 20.3334 22.3625 20.5668 22.6125 20.6334L24.1958 21.0334C25.1625 21.2834 25.4458 22.1334 24.7958 22.9001Z"
+        fill="#1A1A1A"
+      />
+    </svg>
+  );
+};
+export const PeopleIcon = () => {
```
Comment (amjedidiah) [inline src/icons.tsx:112]: Please remove this, as this is not meant for Navbar section

File: src/components/navbar/index.navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import Image from "next/image";
+import Link from "next/link";
+//file import
+import styles from "@/styles/navbar.style";
+import { Links,jumboButtons } from "./constants";
+import ButtonGroup from "../button-group";
+export default function Navbar() {
+  return (
+    <div css={styles}>
+      <section className="bg-shadow">
```
Comment (amjedidiah) [inline src/components/navbar/index.navbar.tsx:11]: This element seems unnecessary. The styles here can be added to the parent element

File: src/styles/navbar.style.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import { Interpolation, Theme, css } from "@emotion/react";
+
+const styles: Interpolation<Theme> = ({
+  isDarkMode,
+  colors: { white, grey2, black },
+  breakpoints,
+}) => css`
+  .bg-shadow{
+        box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.08);
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:9]: Apply `box-shadow` only when fixed to top

File: src/components/navbar/index.navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+/** @jsxImportSource @emotion/react */
+import Image from "next/image";
+import Link from "next/link";
+//file import
+import styles from "@/styles/navbar.style";
+import { Links,jumboButtons } from "./constants";
+import ButtonGroup from "../button-group";
+export default function Navbar() {
```
Comment (amjedidiah) [inline src/components/navbar/index.navbar.tsx:8]: Please make component mobile responsive

File: src/components/navbar/index.navbar.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <div css={styles}>
+      <section className="bg-shadow">
+        <div className="nav-container container">
+          <div className="picture-container">
```
Comment (amjedidiah) [inline src/components/navbar/index.navbar.tsx:13]: Please remove `picture-container`. Use only `Image` and style appropriately

File: src/components/navbar/index.navbar.tsx [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+            <div className="image-container">
+              <Image
+                src="/images/512x512.png"
+                alt="drpassy"
+                width={70}
+                height={70}
+              />
+            </div>
+            <p className="link">Dr. Passy Amaraegbu</p>
```
Comment (amjedidiah) [inline src/components/navbar/index.navbar.tsx:22]: Please remove this text. Error from design

File: src/styles/navbar.style.ts [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    .picture-container {
+      display: flex;
+      gap: 5px;
+      align-items: center;
+
+      .image-container {
+        border-radius: 50%;
+        overflow: hidden;
+
+        img {
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:29]: This style seems unnecessary. Comment out and see

File: src/styles/navbar.style.ts [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+      }
+
+      .link {
+        color: #808080;
+        margin-top: 10px;
+      }
+    }
+    nav {
+      display: flex;
+      justify-content: space-between;
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:43]: This is unnecessary

File: src/styles/navbar.style.ts [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+      gap: 10px;
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:44]: gap is 2rem

File: src/styles/navbar.style.ts [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+      width: 40%;
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:45]: No need for this. Each nav-item has padding of 0.5rem

File: src/styles/navbar.style.ts [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+      font-size: 16px;
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:46]: This is unnecessary as it converts to the default font-size set generally

File: src/styles/navbar.style.ts [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    .link {
+      color: #808080;
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:50]: Styling here in incorrect. Please change

File: src/styles/navbar.style.ts [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    .button-group{
```
Comment (amjedidiah) [inline src/styles/navbar.style.ts:52]: Do better with this styling

File: src/pages/index.tsx (hunk 1 above)
Comment (JumboDaniel) [reply src/pages/index.tsx:21]: @amjedidiah should I remove Jumbo component, I met it there

**Pull request closed without merging**